lib/src/Uncertainty/Model/openturns/DistributionImplementation.hxx

```diff
@@ -876,6 +880,8 @@ protected:
   mutable UniVariatePolynomial generatingFunction_;
 
 #ifndef SWIG
+  //To allow use of wrappers
+  friend class Distribution;
```

Comment:
doesnt seem required as Distribution.getPDF just uses DistributionImplementation.getPDF

Reply:
This was removed, so this conversation can be resolved.

Reply:
Done